### ISSUE TITLE
Feature request: Allow sorting Extensions

### ISSUE THREAD

**Issue by boustanihani** [adobe/brackets#10629]
Could you please allow sorting extensions by the **number of installs** (for example) ?

**Comment by Equinoxdawg** [comment]
From what I remember, [this extension by dnbard](https://github.com/dnbard/brackets-extension-rating) does what you're after and more.

**Comment by zaggino** [comment]
There's an extension called `Extensions Rating` by @dnbard which provides this feature.

**Comment by redmunds** [comment]
Or you can use the http://brackets.dnbard.com/ web site

**Comment by boustanihani** [comment]
Thanks, good to know :)

IMO, such a feature should be a default core extension, don't you think so?

**Comment by AllThingsSmitty** [comment]
@redmunds that's great. Thanks for sharing.

**Comment by peterflynn** [comment]
Yes, we would like to expose this in the core Brackets UI in the future.  One issue is that the extension "download" counts are currently not that reliable: updating an existing extension is recorded as a new download, and occasionally bots will generate thousands of false downloads.  But if we iron out some of those kinks, this would be a great built-in feature.

I see a few existing backlog items somewhat related to this -- e.g. [Extension Ratings and Reviews](https://trello.com/c/UFYZl18d/795-extension-ratings-and-reviews) or [better Extension Download Number Reports](https://trello.com/c/G0AjAo2M/1266-extension-download-number-reports) -- but none that track it directly ...so leaving this issue open for now.

**Comment by ryanstewart** [comment]
I'd like to do a big batch of extension-related improvements sometime in the next few months. There are a number of improvements on both the registry itself and how we expose it in Brackets that would be nice to have, this included.

**Comment by dnbard** [comment]
@ryanstewart does these changes in registry will be backward compatible to registry format that Brackets are using today?
Also is there a list of what changes it will be?
I'm asking because my extension - https://github.com/dnbard/brackets-extension-rating are dependent on Registry information and I want to integrate with them as soon as possible.
